core/cmd/lambda/main.go

```diff
@@ -67,14 +76,14 @@ func (h corsHeaders) setHeaders(resp events.APIGatewayProxyResponse) events.APIG
 
 func handler(
```

Comment:
**suggestion** To increase readability and decrease cognitive overload, I suggest refactoring this function and breaking it down into smaller ones.

Reply:
@diegohordi Good proposal. Please log an issue, we come back to it later if you don't mind. Thanks!

Reply:
@diegohordi please do not hesitate to execute straight away if you feel like so - refactoring is always good when we don't notice invested effort ;)